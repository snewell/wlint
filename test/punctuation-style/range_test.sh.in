I only date people aged 27-32.

Sometimes I like to put extra spaces in my ranges, like this: 52     – 17.

Emdashes aren't good for ranges.  14—24
