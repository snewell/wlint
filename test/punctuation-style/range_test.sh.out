<stdin>-1:24 endash.replace-hyphen
<stdin>-3:62 endash.preceeding-space
<stdin>-3:62 endash.trailing-space
<stdin>-5:34 endash.replace-emdash
